WPMedia/fusion-news-theme-blocks
# TMEDIA 725: Convert Footer block

## Description

Convert Footer block to V2

## Jira Ticket

- [TMEDIA-725](https://arcpublishing.atlassian.net/browse/TMEDIA-725)

## Acceptance Criteria

Footer block is migrated to the new architecture
* Includes Custom fields of:
  * ‘Lazy Load block?’ checkbox - defaulted to false (with existing help text)\

## Test Steps

1. Checkout this branch `git checkout tmedia-725`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/footer-block`
3. Open `https://localhost/pagebuilder/pages`
4. Add Footer block to a test page
5. Configure content:
![image](https://user-images.githubusercontent.com/2287238/187561230-0e805c2f-3641-49ca-a683-8b034311e77f.png)
6. Run Storybook with `npm run storybook`
7. Check Footer Storybook

## Effect Of Changes

### Before


### After

![image](https://user-images.githubusercontent.com/2287238/187716659-1b251bb6-64d2-40a6-9f38-fa4343c940d0.png)


## Dependencies or Side Effects

N/A

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
